src/client/components/Navigation/NavAssessment/Section/NavigationSection.tsx

```diff
@@ -46,13 +47,18 @@ const NavigationSection: React.FC<Props> = (props) => {
   // // On mount check whether the location matches a child path
   useEffect(() => {
     const match = section.subSections.find((subsection) => {
-      const path = BasePaths.Assessment.section(countryIso, assessmentName, subsection.props.name)
+      const path = ClientRoutes.Assessment.Section.getLink({
```

Comment:
On all, when opening a page in a section, current section should be opened and highlighted https://github.com/openforis/fra-platform/issues/1305